nmtwizard/preprocess/prepoperator.py

```diff
@@ -26,12 +26,12 @@ class ProcessType(object):
       """Type of processing pipeline.
 
       Possible values are:
-        * ``TRAINING``
+        * ``SAMPLING``
         * ``INFERENCE``
         * ``POSTPROCESS``
       """
 
-      TRAINING = 0
+      SAMPLING = 0
```

Comment:
I'm not sure about this rename. We could run a pipeline for training without sampling the data beforehand. What do you think?

Reply:
I renamed all to Training, I just wanted it to be consistent.